CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ All notable changes to `Cerberus Python Client` will be documented in this file.
 
 ### Added
 
+###### 04-04-2018
+- Version 0.12.0 - Added support for versioning of secrets in Cerberus.
+
 ###### 02-06-2018
```

Comment:
Is this date correct?  Or should it be 06-02-2018.

Would it help if the Changelog specified the date format?

Reply:
You're right it should be 06-02-2018.

Feel free to specify the format if you like.